### ISSUE TITLE
Invalid session when trying to fetch trade confirmations

### ISSUE THREAD

**Issue by xagotrunk** [Jessecar96/SteamBot#1048]
There seems to be an issue with mobile trade confirmations. 
Whenever my bot runs Bot.AcceptAllMobileTradeConfirmations(); more often than not the error "Invalid session when trying to fetch trade confirmations" is shown

www.steamstat.us is also showing  Steam Community 100.0% Failed Trades quite frequently that could also be related to the problem.

**Comment by FrozenHaxor** [comment]
I confirm this issue. Users are reporting problems with trade confirmations on bot's end.

Exception `Invalid session when trying to fetch trade confirmations` is thrown upon confirmation attempt.

**Comment by scholtzm** [comment]
Yes, indeed they introduced throttling to several old pages including the main trade offers page. Adjusting your settings will fix this.

**Comment by spyfly** [comment]
How frequently can you confirm offers now without getting issues? I had to switch back to the built in confirmations in SteamTradeOffersBot, as calling Bot.AcceptAllMobileConfirmations() barely resulted in an Invalid Session.

**Comment by waylaidwanderer** [comment]
Someone told me that if you wait 10s between fetching the confirmations list, you can avoid being throttled.

I added a commit over on SteamTradeOffersBot to fix this issue, but I'm hesitant to create a pull request for SteamBot because there are probably more elegant ways to code it.

https://github.com/waylaidwanderer/SteamTradeOffersBot/commit/8fb3879a40c0cfbdc5a5abf55901ba5ddca3eeea

**Comment by FrozenHaxor** [comment]
Looks can be deceiving, I'd say go for it as long as it solves the problem ;)
